### PR TITLE
Add missing function execution in streaming example in getting started guide

### PR DESCRIPTION
**Pull request by choonkeat**
followup for https://github.com/hwchase17/langchainjs/pull/691

1. import statement has warning
2. example exits immediately with no output

### CODE REVIEW

File: examples/src/models/chat/chat_streaming_stdout.ts [hunk 1]
```diff
@@ -36,3 +36,5 @@ export const run = async () => {
     ...
     */
 };
+
+run();
```
Comment (nfcampos) [inline examples/src/models/chat/chat_streaming_stdout.ts:40]: The examples are meant to be run with the `yarn example` script, therefore this is not needed

**Review by nfcampos — COMMENTED** [review]
Thanks, one comment

File: examples/src/models/chat/chat_streaming_stdout.ts (hunk 1 above)
Comment (choonkeat) [reply examples/src/models/chat/chat_streaming_stdout.ts:40]: I see. Is my understanding correct that
- the files are meant to be executed through `yarn example` and thus should not contain `run()`
- the content of these files, however, are embedded into docs, e.g. https://js.langchain.com/docs/getting-started/guide-chat#streaming which would not work without `run()`

Do we prefer
1. to keep `run()` absent from the file
2. add a `// run()` along with a comment to uncomment it (I can't come up with a good enough phrasing that feels right for the file & docs)
3. insert the `run()` at docs/docs/getting-started/guide-chat.mdx and docs/docs/getting-started/guide-llm.mdx instead, e.g.

    ```diff
    - <CodeBlock language="typescript">{Example}</CodeBlock>
    + <CodeBlock language="typescript">{Example}
    + run()</CodeBlock>
    ```

(3) perhaps?

Comment (nfcampos) [reply examples/src/models/chat/chat_streaming_stdout.ts:40]: That's a good point. I think my preference would actually be to remove these run() functions from all the examples, as top-level await is allowed in ESM modules we can just run them without that. But that's a larger change

Comment (nfcampos) [reply examples/src/models/chat/chat_streaming_stdout.ts:40]: FYI I've opened this PR https://github.com/hwchase17/langchainjs/pull/841 which makes it possible to declare examples without `function run()`

Comment (choonkeat) [reply examples/src/models/chat/chat_streaming_stdout.ts]: I've added a commit to make use of your work in #841

Comment (choonkeat) [reply examples/src/models/chat/chat_streaming_stdout.ts]: 🤦‍♂️ fixed formatting

**Pull request merged**